src/src/rx_main.cpp

```diff
@@ -568,30 +576,43 @@ void ICACHE_RAM_ATTR TXdoneISR()
 void setup()
 {
     delay(100);
-    Serial.println("ExpressLRS Module Booting...");
 
 #ifdef PLATFORM_STM32
+#if defined(TARGET_R9SLIMPLUS_RX)
+    CRSF_RX_SERIAL.setRx(GPIO_PIN_RCSIGNAL_RX);
+    CRSF_RX_SERIAL.begin(CRSF_RX_BAUDRATE);
+
+    Serial.setTx(GPIO_PIN_RCSIGNAL_TX);
+#else /* !TARGET_R9SLIMPLUS_RX */
     #ifdef USE_R9MM_R9MINI_SBUS
-    HardwareSerial(USART2);
+    //HardwareSerial(USART2); // This is useless call
```

Comment:
Have you checked this is not needed and working on the r9mm?  When it was added, for what ever reason it was needed along with the setTx/Rx calls.

Reply:
No need to even check :)
That is the C++ contructor call and does not effect any of the available `Serials`. 
Correct USART peripheral is selected by using defined TX and RX in `Begin` method according to variant `PeripheralPins.c` definitions.
Fortunately linked ignore it as a unused variable...